VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/individual-participant-status-list/individual-participant-status-list.component.ts

```diff
@@ -120,20 +120,27 @@ export class IndividualParticipantStatusListComponent implements OnInit, OnDestr
             return false;
         }
 
+        const requester = this.getConsultationRequester();
+        if (requester.case_type_group === CaseTypeGroup.OBSERVER || requester.case_type_group === CaseTypeGroup.PANEL_MEMBER) {
+            return false;
```

Comment:
Valentina, this logic should move to the consultation service

Reply:
It's just flag for UI to display link for consultation or hide, it should be on UI